### ISSUE TITLE
SSL: WRONG VERSION NUMBER traceback

### ISSUE THREAD

**Issue by dmorris68** [sabnzbd/sabnzbd#2284]
Been getting this for a long time now (months). Seems like every hour or so (or some regular interval) these are showing up in the SAB web GUI. Seems to be benign in terms of functionality, but really dirties up the UI and looks alarming. Looks like same issue reported in #820 like 5 years ago.

Running 3.6.1-0ubuntu1~jcfp1~20.04 package on Ubuntu 20.04 LTS.
```
ERROR an hour ago [31/Aug/2022:20:42:52] ENGINE Error in HTTPServer.tick
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cherrypy/wsgiserver/__init__.py", line 2021, in start
    self.tick()
  File "/usr/lib/python3/dist-packages/cherrypy/wsgiserver/__init__.py", line 2090, in tick
    s, ssl_env = self.ssl_adapter.wrap(s)
  File "/usr/lib/python3/dist-packages/cherrypy/wsgiserver/ssl_builtin.py", line 66, in wrap
    s = self.context.wrap_socket(sock,do_handshake_on_connect=True,
  File "/usr/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/usr/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)
```

**Comment by thezoggy** [comment]
something hitting your server using outdated ssl, so prob a probe. you should block the ip / dont expose your sab externally / use reverse proxy / etc

**Comment by Safihre** [comment]
No the problem here is that you are using an old version of CherryPy. As you can see from the linked issues in #820, it was solved in CherryPy/cheroot in 2017: https://github.com/cherrypy/cheroot/pull/32
I think this is because for Ubuntu 20 there is just no more recent version available, right @jcfp ?

Indeed something is pinging your SABnzbd instance. If you find the source, you could kill it there.

**Comment by jcfp** [comment]
@Safihre  Indeed cherrypy 18.x was only packaged after 20.04 came out, so that's still using 8.9.x. These errors don't usually show up with normal use and only get triggered by script kiddies or one of those port scanning ""services"" trying (known vulnerabilities in) ancient ssl versions.

**Comment by dmorris68** [comment]
I saw the cherrypy issues but the error message was different on the ones I saw, so wasn't sure this was the same thing.

SAB is behind an nginx proxy already. Or at least it used to be, LOL. I'm not at home right now to confirm, but recently went through a basement remodel and relocated all my network infrastructure. During the transition, I may have kept SAB alive during the transition by opening a port on the router, and possibly forgot to close it once nginx was back up and running. When I get home tonight I'll verify, so thanks for that reminder.

I've been planning to upgrade 20.04 to 22.04, but I saw someone posting another cherrypy error on 22.04 so I was assuming it was still "broken" on 22.04 as well.
